gallery.html

```diff
@@ -1,25 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="galleryapp.css">
-</head>
+    <link rel="stylesheet" href="galleryapp.css" />
+  </head>
 
-<body>
+  <body>
+    <h1>Random Gallery</h1>
+    <h2>By Ismael Ladejobi</h2>
 
+    <div class="links">
```

Comment:
These links should probably live in a `nav` tag instead of a `div` as that represents navigation.